cluster/manifests/kube-proxy/configmap.yaml

```diff
@@ -23,7 +23,10 @@ data:
       tcpCloseWaitTimeout: 1h0m0s
       tcpEstablishedTimeout: 24h0m0s
     enableProfiling: false
-    featureGates: "ExperimentalCriticalPodAnnotation=true,TaintBasedEvictions=true,PodPriority=true"
+    featureGates:
+      ExperimentalCriticalPodAnnotation: true
```

Comment:
This is no longer needed since we've switched to priorities. Or am I missing something?

Reply:
True, not needed. Removed it!